### DISCUSSION TITLE
Subpixel rendering support for text rendering

### DISCUSSION THREAD

**Discussion by qiangbro** [sumatrapdfreader/sumatrapdf#2538] [Ideas]
Any idea for adding support for **ClearType** or **Subpixel rendering** or something?
it improves the readability of text on existing LCDs (Liquid Crystal Displays), such as laptop screens, Pocket PC screens, and flat panel monitors
![mmm](https://user-images.githubusercontent.com/4646469/167106670-722c9340-da76-4918-a6f8-47f049a54e44.gif)
![sbs](https://user-images.githubusercontent.com/4646469/167106675-075d02ff-6d07-45f9-9553-9131cafa707d.gif)

**Comment by kjk** [comment]
We use mupdf for PDF rendering. Changing how rendering works is hard and beyond the scope of my work.
